#subsystem_diagnostic
uint8[] Diagnostic_Type
uint8[] Level
